Readme.md

```diff
@@ -265,6 +265,16 @@ TIPS
    - Combines many JSON files into a single file
    - Builds a HTML report from JSON with support for debug msgs & embedded Base64 images.
 
+### Spring
```

Comment:
I'd prefer to not copy their readme, just link to it so it does not get out of date

Reply:
Ah good point! I've updated the README to be more minimal and just include a link to `spring-commands-parallel-tests`